BackendApartmentReservation/BackendApartmentReservation/Repositories/Checklist/IApartmentRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BackendApartmentReservation.Database.Entities.Amenities;
+
+namespace BackendApartmentReservation.Repositories.Checklist
+{
+    public interface IApartmentRepository
+    {
+        Task<DbApartmentAmenity> CreateApartmentAmenity(string address);
```

Comment:
All of those interfaces are not registered in IoC config

Reply:
fixed